packages/edit-widgets/src/components/header/document-tools/index.js

```diff
@@ -75,14 +75,11 @@ function DocumentTools() {
 		[ setIsListViewOpened, isListViewOpen ]
 	);
 
-	// If there's a block toolbar to be focused, disable the focus shortcut for the document toolbar.
-	const { showFixedToolbar } = useShowBlockTools();
-
 	return (
 		<NavigableToolbar
 			className="edit-widgets-header-toolbar"
 			aria-label={ __( 'Document tools' ) }
-			shouldUseKeyboardFocusShortcut={ ! showFixedToolbar }
+			shouldUseKeyboardFocusShortcut={ ! useHasBlockToolbar() }
```

Comment:
I'd be minded to move the hook call to a `const`. It's a more standard pattern and also avoids issues with calling hooks conditionally in the future.